Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[workspace]
+
+members = [
+    "examples/abitest/abitest_common",
+    "examples/abitest/module_0/rust",
+    "examples/abitest/module_1/rust",
+    "examples/abitest/tests",
+    "examples/chat/module/rust",
+    "examples/hello_world/module/rust",
+    "examples/machine_learning/module/rust",
+    "examples/private_set_intersection/module/rust",
+    "examples/running_average/module/rust",
+    "examples/rustfmt/module/rust",
+    "examples/translator/common",
+    "examples/translator/module/rust",
+
+    "oak/server/rust/oak_abi",
+    "oak/server/rust/oak_runtime",
+
+    "sdk/rust/oak",
+    "sdk/rust/oak_derive",
+    "sdk/rust/oak_log",
+    "sdk/rust/oak_tests",
+    "sdk/rust/oak_utils",
+]
+
+# Patch dependencies on oak crates so that they refer to the versions within this same repository.
+#
+# From https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html :
+# > Cargo doesn’t assume that crates in a workspace will depend on each other, so we need to be
+# > explicit about the dependency relationships between the crates.
+[patch.crates-io]
+# Oak.
+asylo_rust = { path = "oak/asylo_rust" }
+oak = { path = "sdk/rust/oak" }
+oak_abi = { path = "oak/server/rust/oak_abi" }
+oak_derive = { path = "sdk/rust/oak_derive" }
+oak_log = { path = "sdk/rust/oak_log" }
+oak_runtime = { path = "oak/server/rust/oak_runtime" }
+oak_tests = { path = "sdk/rust/oak_tests" }
+oak_utils = { path = "sdk/rust/oak_utils" }
+runner = { path = "runner" }
```

Comment:
`runner` should  also be part of the workspace itself, right? (even though it's not really used yet)

Reply:
Done